src/frontend/src/App.vue

```diff
@@ -9,3 +9,13 @@ import { Vue } from "vue-property-decorator";
 
 export default class App extends Vue {}
 </script>
+
+<style lang="scss">
+@import "./style.scss";
+
+body {
```

Comment:
Can you add the new font style in this PR too please:
```
@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;400&display=swap");
body {
  font-family: "Montserrat", sans-serif;
}
```

Reply:
Added!